### ISSUE TITLE
Flaky test: Bullet clears the font color after deleting all text

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#4713]
```
FAIL   puppeteer-e2e  src/e2e/puppeteer/__tests__/color.ts > Bullet clears the font color after deleting all text and moving the cursor away
TimeoutError: Waiting for selector `[aria-label="text color swatches"] [aria-label="red"]` failed
 ❯ Function.waitFor node_modules/puppeteer-core/lib/esm/puppeteer/common/QueryHandler.js:211:46
 ❯ CdpFrame.waitForSelector node_modules/puppeteer-core/lib/esm/puppeteer/api/Frame.js:541:21
 ❯ CdpPage.waitForSelector node_modules/puppeteer-core/lib/esm/puppeteer/api/Page.js:1387:20
 ❯ click src/e2e/puppeteer/helpers/click.ts:34:9
     32|   const nodeHandle =

     33|     typeof nodeHandleOrSelector === 'string'
 Test Files  1 failed | 33 passed | 3 skipped (37)
      Tests  1 failed | 169 passed | 12 skipped (182)
     34|       ? await waitForSelector(nodeHandleOrSelector)
       |         ^
   Start at  17:02:09
     35|       : nodeHandleOrSelector.asElement()
   Duration  175.75s (transform 4.50s, setup 557ms, import 9.74s, tests 460.41s, environment 44.46s)

     36| 
 ❯ src/e2e/puppeteer/__tests__/color.ts:91:3
```

https://github.com/cybersemics/em/actions/runs/30027492611/attempts/1
